content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -175,7 +175,7 @@ For more information on version skews, see:
 3. Add the Kubernetes `apt` repository:
 
    ```shell
-   echo "deb [signed-by=/usr/share/keyrings/kubernetes-archive-keyring.gpg] https://apt.kubernetes.io/ kubernetes-xenial main" | sudo tee /etc/apt/sources.list.d/kubernetes.list
+   echo "deb [signed-by=/usr/share/keyrings/kubernetes-archive-keyring.gpg] http://packages.cloud.google.com/apt/ kubernetes-xenial main" | sudo tee /etc/apt/sources.list.d/kubernetes.list
```

Comment:
Small alteration in the link, but otherwise lgtm 

https://packages.cloud.google.com/apt/ kubernetes-xenial main